### PR TITLE
Add common aliases

### PR DESCRIPTION
**Pull request by shyamg28**
The aliases added are extremely common knowledge and commonly used by now. These could be added to the cheatsheet to provide value add for everyone.

### CODE REVIEW

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Comment by Ritikaa96** [comment]
I see the Netlify build failed due to timeout. you could remove it by triggering the build again after committing new changes or using
` git commit --amend --no-edit && git push --force-with-lease`

**Comment by sftim** [comment]
> I see the Netlify build failed due to timeout. you could remove it by triggering the build again after committing new changes or using ` git commit --amend --no-edit && git push --force-with-lease`

Actually, our [advice](https://lwkd.info/2023/20230606#developer-news) is to rebase against _main_.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -88,6 +89,21 @@ kubectl config unset users.foo                       # delete user foo
 # short alias to set/show context/namespace (only works for bash and bash-compatible shells, current context to be set before using kn to set namespace) 
 alias kx='f() { [ "$1" ] && kubectl config use-context $1 || kubectl config current-context ; } ; f'
 alias kn='f() { [ "$1" ] && kubectl config set-context --current --namespace $1 || kubectl config view --minify | grep namespace | cut -d" " -f6 ; } ; f'
+
+# the following aliases can be used to run common commands faster.
```
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/en/docs/reference/kubectl/cheatsheet.md:96]: @shyamg28 Please could you advise why this is featured in the comments section as an addendum to the section, Kubectl context and configuration?

If we are to include common aliases, would it not be more prudent to have a section of its own within this document?

**Comment by Ritikaa96** [comment]
Hi @shyamg28 , Are you still working on this PR?
Please rebase if you are. or else we can close this PR.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -100,6 +101,21 @@ kubectl config unset users.foo                       # delete user foo
...
+alias kd='kubectl describe'
+alias kg='kubectl get -owide'
+alias ka='kubectl apply -f'
+alias kex='kubectl exec -it'
+alias ke='kubectl edit'
+export dry='--dry-run=client -oyaml'
+export y='-oyaml'
+export w='-owide'
+export j='-ojson'
+export force='--force --grace-period=0'
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/cheatsheet.md:118]: These alias are aimed for productivity IIUC.
However, they are also dangerous when users are running Shell scripts.
Having `ka`, `kd`, and even `w`, `y`, `dry` defined is not always a good thing.
I'd suggest we leave these optimizations to users themselves.

Comment (sftim) [reply content/en/docs/reference/kubectl/cheatsheet.md:118]: :+1:

We could _blog_ about these aliases. I don't think they're a good fit for the _cheat sheet_ page.

**Pull request closed without merging**